development/test-log-collector/pkg/config/config_test.go

```diff
@@ -132,7 +132,7 @@ func TestLoadDispatchingConfig(t *testing.T) {
   channelID: "chanID1"
   onlyReportFailure: false
   testCases:
-    - "rafter"
+    - "busola-migrator"
```

Comment:
Why are we replacing this?

Reply:
well.. this is just a config test that was mentioning rafter as an example..
If the config for rafter is being removed I thought its not a good case for this test case so I replaced it with sth else